content/blog/newsletter-006/index.md

```diff
@@ -1,6 +1,9 @@
 +++
 title = "This Month in Rust GameDev #6 - January 2020"
 date = 2020-02-16
+description = "The start of the new year hails the arrival of a slur of new content updates, job openings, and released talks. Dig into this sixth issue of the Rust GameDev Workgroup's monthly newsletter for your dose of Rust gamedev goodieness."
```

Comment:
Is it possible to use multiline strings here to comply with the 80 chars limit?

Reply:
I'll check to see if that still displays correctly